src/main/java/org/example/App.java

```diff
@@ -1,8 +1,21 @@
 package org.example;
 
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.stream.Collectors;
 
-    public class App {
+public class App {
     public static void main( String[] args ) {
-        System.out.println( "Hello World!" );
+        System.out.println("Sorted word list: ");
+        sortedList(args).forEach(x -> System.out.println(x));
+    }
+
+    /**
+     * Sort array of words
+     * @param str Array of words which must be sorted
+     * @return List of sorted words
+     */
+    public static ArrayList<String> sortedList(String[] str){
+        return (ArrayList<String>)Arrays.asList(str).stream().sorted().collect(Collectors.toList());
```

Comment:
You have to depend on abstractions `List<String>` not on direct implementations, `ArrayList<String>`. See SOLID principles. https://en.wikipedia.org/wiki/SOLID, please make some notes regarding that in Notion.

[Dependency inversion principle](https://en.wikipedia.org/wiki/Dependency_inversion_principle)



All mentioned above is just general recommendations. The general idea is while working with Collections in Java you should return general interfaces from the methods, e.g. Map<String, String>, List<String>, Set<String>, etc.